WcaOnRails/db/seeds/development/competitions.seeds.rb

```diff
@@ -153,32 +156,17 @@ def random_wca_value
       latitude_degrees: rand(-90.0..90.0),
       longitude_degrees: rand(-180.0..180.0),
     )
+    events = random_events
+    competition.events = events
```

Comment:
In one of the 3 "create competition" sections it is redundant but I thought I'd keep the structure consistent. I'm happy to change it if you feel it's better.